spock-core/core.gradle

```diff
@@ -13,6 +13,7 @@ Spock is inspired from JUnit, jMock, RSpec, Groovy, Scala, Vulcans, and other fa
 dependencies {
   compile libs.groovy // easiest way to add Groovy dependency to POM
   compile libs.junit
+  compile libs.annotations
```

Comment:
The dependency only contains annotation declarations (http://grepcode.com/file/repo1.maven.org/maven2/org.jetbrains/annotations/13.0/org/intellij/lang/annotations), I think the added value superseeds the cost.

If you don't agree, I could still add only the `Language.java` file to the project (with the original package) and change the `RetentionPolicy` to `SOURCE`.
However, I think that's a hacky solution!